css/helper.css

```diff
@@ -12,3 +12,8 @@
     gap: 2rem;
 }
 
+.grid-75-25 {
+    grid-template-columns: 60% 1fr;
```

Comment:
You can instead define a new class, given that the name and property being defined dont match
like `grid-60-40`

Reply:
yes it is indeed a new class but as I was experimenting, at the end  forgot to change it to 60-40